service/dap/server.go

```diff
@@ -748,6 +750,12 @@ func cleanExeName(name string) string {
 }
 
 func (s *Server) onLaunchRequest(request *dap.LaunchRequest) {
+	if s.debugger != nil {
+		s.sendErrorResponse(request.Request, FailedToLaunch,
+			"Failed to launch", "debugger already started - use attach request to connect")
```

Comment:
We have had conversations about "remote" vs "connect" or "external", but I don't think we ever settled on an alternative. And then since there was a general push for consistency with the legacy configurations, I figured we might as well keep "remote". We can have a chat about this again.

A misbehaving client that sends duplicate messages is not a viable client that can ensure a valid debug session. I would rather tailor error messages to cases that the users can run into and control. In case they switch from launch+host/port to attach+remote+host/port and difference types of server commands, I wanted to make sure they understand the difference. Also in case somebody tries to use deprecated launch+remote. After your question, I actually think we need more details not less. How about?
```
debugger already started - use remote attach to connect to a server with an active debug session
```

Reply:
> debugger already started - use remote attach to connect to a server with an active debug session

This seems like a more helpful message since it explains what we are assuming the user wants to do (and if they don't want to do that it will be easier for them to notice that something else is wrong).

Reply:
Done

Reply:
avoiding further change on user's configuration setting is a good argument because that's the whole point of this project. Let's keep `remote` mode.